package.json

```diff
@@ -2,15 +2,14 @@
   "name": "react-components",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
-  "sideEffects": false,
```

Comment:
do not remove this line it's important for the tree-shacking

Reply:
This was removed by merge conflict, I'll add it back.